textdb/textdb-textql/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>edu.uci.ics.textdb</groupId>
+    <artifactId>textdb</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>textdb-textql</artifactId>
+  <name>textdb-textql</name>
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+  <dependencies>
+    <dependency>
+        <groupId>edu.uci.ics.textdb</groupId>
+        <artifactId>textdb-dataflow</artifactId>
+        <version>${textdb.version}</version>
+    </dependency>
+	<dependency>
+	    <groupId>org.apache.commons</groupId>
+	    <artifactId>commons-lang3</artifactId>
+	    <version>3.0</version>
+	</dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>javacc-maven-plugin</artifactId>
```

Comment:
Add comments to the codebase to explain how mvn and javacc work together using this `javacc-maven-plugin` plugin.